.github/workflows/run-tests-windows.yml

```diff
@@ -121,11 +121,19 @@ jobs:
         run: clang --version
 
       - name: Run sandbox
-        # Set heap limits manually, there seams to be bug in detection of avaiable memory
-        # which results in undefined behavior at runtime
+        # Set heap limits manually, there seams to be bug in detection of available memory
+        # which results in undefined behaviour at runtime
         env:
-          SCALANATIVE_MIN_SIZE: 128M
-          SCALANATIVE_MAX_SIZE: 1G
+          SCALANATIVE_MAX_SIZE: 128M
```

Comment:
I guess all of this will disappear once rebased on top of #2293 ?

Reply:
Yes, that's correct. I've create that fix separately so it can be applied at any time (eg. before merging PR with CodeGen support for Windows)